docs/source/06_nodes_and_pipelines/04_micro_packaging.md

```diff
@@ -1,17 +1,20 @@
+```eval_rst
+.. warning::  `kedro pipeline package/pull` will be deprecated in 0.18.0. We recommend using `kedro micropkg package` instead.
+```
 
 # Micro-packaging
 
-Micro-packaging allows users to share Kedro pipelines across codebases, organisations and beyond.
+Micro-packaging allows users to share Kedro micro-packages across codebases, organisations and beyond.
```

Comment:
I wonder if we should elaborate a bit more on what a "Kedro micro-package" is. This might come as new terminology to some users.

Reply:
I thought it was self-explanatory, apart from it being a simple way of reusing code across projects what else could we say?

Reply:
As far as I know micro-packaging is new terminology within Kedro (keep my honest here @lorenabalan), so I can imagine people might be wondering what's the difference between packaging `pipelines` and `micro-packages`. I'd just add a line explaining that a micro-package is any part of the Python code of a Kedro project, which includes pipelines.

Reply:
Agreed - I don't think micropackage is standard terminology that people will be familiar with. I'd explicitly call out "utility functions" in addition to pipelines as examples of what could constitute a micropackage.